### PR TITLE
[#157190168] Cleanup manifests following cf-deployment migration (part 2)

### PR DESCRIPTION
**Pull request by alext**
What
----

This is the second part of cleaning up the manifests following the cf-deployment migration. Part 1 (#1350) covered all the changes that would still leave a no-op deploy. The changes in this PR will cause a redeploy of all the VMs etc.

This PR covers a bunch of cleanup to bring our manifests closer to the cf-deployment manifests. See the individual commits for details of the changes.

How to review
-------------

Code review is probably easiest commit-by-commit.

* Deploy this branch on top of a deploy from master
* Check everything looks good.
* Rotate the leaf certificates (step 3 [here](https://team-manual.cloud.service.gov.uk/team/rotating_credentials/#rotating-certificates)), but **not the CA certs** (rotating CA certs is currently know to cause downtime)
* Redeploy from this branch with the rotated certs
* Verify all is good.

Who can review
--------------

Not @bandesz or myself.

### CODE REVIEW

**Comment by keymon** [comment]
Running the tests I got a error like:

```
	
• Failure [106.844 seconds]
	
[v3]
/tmp/build/f541ec31/src/github.com/cloudfoundry/cf-acceptance-tests/cats_suite_helpers/cats_suite_helpers.go:206
  v3 buildpack app lifecycle
  /tmp/build/f541ec31/src/github.com/cloudfoundry/cf-acceptance-tests/cats_suite_helpers/cats_suite_helpers.go:212
    with a java_buildpack
    /tmp/build/f541ec31/src/github.com/cloudfoundry/cf-acceptance-tests/v3/app_lifecycle.go:124
      can run spring apps [It]
      /tmp/build/f541ec31/src/github.com/cloudfoundry/cf-acceptance-tests/v3/app_lifecycle.go:130
      Timed out afis cater 60.045s.
      Expected
          <string>: 404 Not Found: Requested route ('CATS-3-APP-99e9a8451be5548c.hector.dev.cloudpipelineapps.digital') does not exist.
      to contain substring	
          <string>: ok
      /tmp/build/f541ec31/src/github.com/cloudfoundry/cf-acceptance-tests/v3/app_lifecycle.go:150
```

This kind of error might be due problems registering the route. Maybe route emitter? I will have a look around that.

**Comment by keymon** [comment]
A second rerun of the tests pass clean :thinking: . I am checking the services around route emitter, anyway.

**Comment by keymon** [comment]
> A second rerun of the tests pass clean thinking . I am checking the services around route emitter, anyway.

I did not get any problem with this. All the test run fine several times. 

I would merge, just a pair of comments in https://github.com/alphagov/paas-cf/pull/1354#discussion_r188659330 and https://github.com/alphagov/paas-cf/pull/1354#issuecomment-389559390

File: manifests/cf-manifest/cloud-config/operations/060-cdn-broker.yml [hunk 1]
```diff
@@ -1,19 +1,11 @@
 ---
-# FIXME: Remove these vm types by using extensions instead
```
[the remark this answers is not in the document]
Comment (alext) [reply manifests/cf-manifest/cloud-config/operations/060-cdn-broker.yml]: I don't think that would gain anything as it's using an EC2 instance type that's not used by any other VMs, so we'd still need a base `vm_type` for it.

**Comment by alext** [comment]
> I think it is safe to also remove all the operations on the webdav attributes for the blobstores.

That's a good point. I'll take a look into that. Given this doesn't relate to a change being made in this PR (instead it's an additional thing we can be doing), I'll add it to the part 3 PR (#1358) so that it doesn't block this one.

**Pull request merged**